USAGE.md

```diff
@@ -1,370 +1,707 @@
 # Introduction
-This document describes how to query and manipulate ATT&CK data in this repository. Machine-readable ATT&CK data is currently available via two mechanisms:
-* a JSON-based [STIX 2.0](https://oasis-open.github.io/cti-documentation/stix/intro) format.
-* a [TAXII Server](https://medium.com/mitre-attack/att-ck-content-available-in-stix-2-0-via-public-taxii-2-0-server-317e5c41e214) hosting the same STIX 2.0 content
 
-The MediaWiki API was deprecated with the ATT&CK content release in October 2018 and fully removed on February 12th, 2019. Luckily, STIX 2.0 is just JSON and so should be very accessible from Python and other programming languages. If you are using Python, the [python-stix2](https://github.com/oasis-open/cti-python-stix2) library can help you work with the content as shown in the examples below. 
+This document describes how to query and manipulate ATT&CK data in this repository. It is divided into three sections:
 
-# Mapping Concepts
-First, we must describe how ATT&CK objects and properties map to STIX 2.0 objects and properties.
+- [The ATT&CK Data Model](#The-ATT&CK-Data-Model), which describes the format of the data and highlights how it departs from the stock STIX2.0 format
+- [Accessing ATT&CK Data in Python](#Accessing-ATT&CK-Data-in-Python), which describes different methodologies that can be used to load the ATT&CK data into a script
+- [Python Recipes](#Python-Recipes), which provides python3 examples of common ways to query the ATT&CK data once loaded
 
-## Objects
-In ATT&CK, there are three main concepts (excluding Tactics for now): Techniques, Groups, and Software. Most techniques also have Mitigations. STIX 2.0 describes these as objects and uses different terminology to describe them. The following table is a mapping of ATT&CK concepts to STIX 2.0 objects:
+This document does not describe the generic STIX format, instead detailing how said format applies to ATT&CK and where ATT&CK departs from that format. To find out more about STIX, please see [the STIX 2.0 website](https://oasis-open.github.io/cti-documentation/stix/intro). 
```

Comment:
This is personal preference, but is it possible to make the links open in a new tab? For me, it feels like a disruption if it changes my current tab and then I have to go back, etc. But I'm sure there are people out there who prefer it to open in the same tab, so. Just a thought!

Reply:
I don't think it's easily possible to do this in markdown. You can always do a middle-mouse-button click which opens a new tab in most browsers though 🤷‍♀️